TheForceEngine/TFE_FrontEndUI/modLoader.cpp

```diff
@@ -444,6 +451,21 @@ namespace TFE_FrontEndUI
 			}
 			filterMods(s_viewMode != VIEW_FILE_LIST);
 		}
+
+		ImGui::SameLine(510.0f * uiScale);
+		if (ImGui::Button("Refresh Mod Listing"))
+		{
+			s_modsRead = false;
+			modLoader_read();
+		}
+
+		#ifdef _WIN32
```

Comment:
use osShellExecute() from system.h, that way you don't need to include windows.h here.

Reply:
Switched to osShellExecute